plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -205,7 +205,7 @@ fu_uefi_dbx_device_probe(FuDevice *device, GError **error)
 		return FALSE;
 	if (!fu_firmware_parse_bytes(kek, kek_blob, 0x0, FWUPD_INSTALL_FLAG_NONE, error))
 		return FALSE;
-	fu_device_add_instance_strup(device, "ARCH", EFI_MACHINE_TYPE_NAME);
+	fu_device_add_instance_strup(device, "ARCH", fu_uefi_dbx_get_efi_arch());
```

Comment:
If you don't get back a value from `fu_uefi_dbx_get_efi_arch()` presumably you don't want to add an instance.  Should we fail probe in this case?